topics/variant-analysis/tutorials/sars-cov-2-variant-discovery/tutorial.md

```diff
@@ -4,7 +4,6 @@ layout: tutorial_hands_on
 title: Mutation calling, viral genome reconstruction and lineage/clade assignment from SARS-CoV-2 sequencing data
 subtopic: one-health
 level: Intermediate
-zenodo_link: "https://zenodo.org/record/5036687"
```

Comment:
is this record not valid or relevant anymore?

Reply:
the current version of the tutorial uses data from the ENA, which we never copied to zenodo

Reply:
and the zenodo record, conversely, has data that was only used previously